src/main/java/io/managed/services/test/TestUtils.java

```diff
@@ -345,4 +345,14 @@ public void checkServerTrusted(java.security.cert.X509Certificate[] certs, Strin
 
         return context;
     }
+
+    public static boolean inRange(Double expected, Double actual, Double allowedDifferencePercentage) {
```

Comment:
is this function used anywhere? If not we can remove it.

Reply:
not used will remove